src/components/BarChart/BarChart.jsx

```diff
@@ -280,13 +280,15 @@ const BarChart = createClass({
 			...marginOriginal,
 		};
 
+		const svgClasses = cx(className, '&');
+
 		// TODO: Consider displaying something specific when there is no data,
 		// perhaps a loading indicator.
-		if (_.isEmpty(data)) {
+		if (_.isEmpty(data) || width < 1 || height < 1) {
```

Comment:
I added these just to guard against the case where charts might be rendered with 0 or negative widths/heights.